docs/LabVIEW_client.md

```diff
@@ -13,7 +13,7 @@
 
 2. Download VeloView from this [website](https://www.paraview.org/download/).
 
-3. Download the monoDrive Simulator from [here](https://lumen.ni.com/nicif/US/GB_EVALTLKTLVMONODRIVE/content.xhtml).
+3. Download the monoDrive Simulator from [here](https://www.monodrive.io/register).
```

Comment:
Where is this document linked on the left panel?

Reply:
It's not linked on the left panel, and it's not being used anymore. Removing this file.